crates/cext/cbindgen.toml

```diff
@@ -1,19 +1,24 @@
 include_version = true
-style = "type" 
+style = "type"
 sys_includes = ["complex.h"]
 after_includes = """
+#ifdef QISKIT_C_PYTHON_INTERFACE
+    #include<Python.h>
+#endif
```

Comment:
ludicrously minor: `#include <Python.h>` wants a space between the `include` and the `<` haha.

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/13934/commits/b92f156d9293655df2757ff11e9c8696da544185